docs/create-stack.md

```diff
@@ -65,7 +65,9 @@ Note that serverless applications, by default, do not run in a VPC. You can, how
 
 ### Add a Database
 
-Click **Enable** to create an Amazon RDS Postgres or MySQL database for your account. Use the settings option to select an existing database instead and to configure other database-related options. 
+Click **Enable** to create an Amazon RDS Postgres database, an Amazon RDS MySQL database, or a Redis in-memory database for your account. Use the settings option to select an existing database instead and to configure other database-related options. 
```

Comment:
maybe a line here to mention customers can add additional dbs of diff types on the update after the stack's up?

Reply:
@zsimjee resolved by recent commit.